cmd/otel-allocator/main.go

```diff
@@ -18,10 +21,20 @@ import (
 	lbdiscovery "github.com/open-telemetry/opentelemetry-operator/cmd/otel-allocator/discovery"
 	allocatorWatcher "github.com/open-telemetry/opentelemetry-operator/cmd/otel-allocator/watcher"
 	ctrl "sigs.k8s.io/controller-runtime"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
 var (
-	setupLog = ctrl.Log.WithName("setup")
+	setupLog     = ctrl.Log.WithName("setup")
+	httpDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "allocator_http_duration_seconds",
+		Help: "Duration of HTTP requests.",
+	}, []string{"path"})
+	events = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Do not use `events` here, use for example `eventsMetric` to prevent variable collisions and confusion.